lib/ansible/modules/network/junos/junos_netconf.py

```diff
@@ -139,7 +139,7 @@ def map_params_to_obj(module):
     for key, value in iteritems(obj):
         # validate the param value (if validator func exists)
         validator = globals().get('validate_%s' % key)
-        if all((value, validator)):
+        if callable(validator):
```

Comment:
I think you still need `and value` here

And while we're at it, I don't really see a reason for validator, as the only one is netconf_port, so one might as well fold that in to this method and make the check `if key == 'netconf_port' and value:` right? (Or if you really want room for more validators, `if not value: continue` followed by checking the key)